spec/features/answer/edit_spec.rb

```diff
@@ -17,14 +17,16 @@
     end
 
     scenario 'edits his answer' do
-      within '.answers' do
+      within "#answer_#{answer.id}" do
         click_on 'Edit'
         fill_in 'Body', with: 'Edited Answer'
         click_on 'Save'
+        sleep 1
```

Comment:
not good practice) it is ok for now, but for prod it is not accessible
there is should be smth like  
```
scenario 'edits his answer', js: true
```
which will wait loading js before do next action